### PR TITLE
Create gdcr22-krakow

### PR DESCRIPTION
**Pull request by 3v0k4**
Will add more details later on :)

### CODE REVIEW

File: _data/events/gdcr22-krakow [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
```
Comment (rradczewski) [inline _data/events/gdcr22-krakow:1]: Needs to end in .json so we process it 😊

**Review by rradczewski — CHANGES_REQUESTED** [review]

File: _data/events/gdcr22-krakow (hunk 1 above)
Comment (3v0k4) [reply _data/events/gdcr22-krakow]: ✅ Sorry, my bad!

**Comment by rradczewski** [comment]
Whoops, the preview suggests that you found a bug @3v0k4 😬 I'll have a look later, but thanks for fixing the file extension so quickly!🤗

**Comment by 3v0k4** [comment]
I'm real sorry. Let me know if I can help!

**Comment by rradczewski** [comment]
Appreciate the edge-case :) fixed in dd1c514 – turns out we didn't have an appropriate null-check for all places where we use the moderators list.

**Pull request merged**